setup.py

```diff
@@ -59,7 +51,8 @@ def _get_data_files():
 
 setuptools.setup(
     name=NAME,
-    version=_get_version(),
+    # There is no way that can be right
```

Comment:
lol, this might seem a bit cryptic in the future - and might be intepreted as a "FIXME" comment. Maybe make it more clear that this is intentional

Reply:
Sure, but then I lose the joke - it's Nappa's response in the scene.
Will add relevant information of course.